Makefile

```diff
@@ -147,6 +147,8 @@ check: compile-cairo compile-starknet
 deps: check-python-version build-cairo-2-compiler build-cairo-1-compiler
 	cargo install flamegraph --version 0.6.2
 	cargo install cargo-llvm-cov --version 0.5.14
+	pyenv install -s pypy3.9-7.3.9
+	pyenv install -s 3.9.15
```

Comment:
If we add this here, we need it for `deps-macos` too.

Reply:
Fixed.